packages/medusa/src/services/gift-card.js

```diff
@@ -103,9 +103,11 @@ class GiftCardService extends BaseService {
         .where(where)
         .andWhere(
           new Brackets(qb => {
-            qb.where(`gift_card.code ILIKE :q`, {
-              q: `%${q}%`,
-            }).orWhere(`order.display_id = :d`, { d: q })
+            let brackets = qb.where(`gift_card.code ILIKE :q`, { q: `%${q}%` })
```

Comment:
Can we do as `orderService.listAndCount` : https://github.com/medusajs/medusa/blob/44d31b4d833b602e20ecac4ce2b8b949d3d22874/packages/medusa/src/services/order.js#L236?